app/src/main/java/com/example/linkdevmvvm/ui/base/BaseViewModel.java

```diff
@@ -0,0 +1,58 @@
+package com.example.linkdevmvvm.ui.base;
+
+import android.arch.lifecycle.ViewModel;
+import android.databinding.ObservableBoolean;
+
+import com.example.linkdevmvvm.App;
+import com.example.linkdevmvvm.service.ServicesInterface;
+
+import javax.inject.Inject;
+
+import retrofit2.Call;
+
+public abstract class BaseViewModel extends ViewModel {
+
+    private final ObservableBoolean mIsLoading = new ObservableBoolean();
+    private final ObservableBoolean mIsRefreshing = new ObservableBoolean();
+    @Inject
+    ServicesInterface servicesInterface;
+    private Call call;
```

Comment:
This should be a list of calls, to handle the case if there is more than one call in the ViewModel

Reply:
I removed the Call object because livedata handle auto lifecycle "if the view destroyed it won't observe the changes "